whois-api/src/test/java/net/ripe/db/whois/api/mail/dequeue/MessageServicePasswdErrorTestIntegration.java

```diff
@@ -0,0 +1,159 @@
+package net.ripe.db.whois.api.mail.dequeue;
+
+import jakarta.mail.MessagingException;
+import net.ripe.db.whois.update.mail.MailSenderStub;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@Tag("IntegrationTest")
+public class MessageServicePasswdErrorTestIntegration extends AbstractMailMessageIntegrationTest {
+
+    @Autowired
+    private MailSenderStub mailSenderStub;
+
+
+    private static final String FORMATTED_PASSWORD_ERROR = """
+            ***Error:   Password authentication is not allowed in Mailupdates, because your
```

Comment:
Why not use UpdateMessages.getText?

Reply:
We discussed this already
UpdateMessages.getText doesn't give you the correct format
so it will never match.
The format of the output and what UpdateMessages.getText or getFormattedText is not the same